fastplotlib/graphics/_features/_colors.py

```diff
@@ -253,7 +251,7 @@ class CmapFeature(ColorFeature):
     """
 
     def __init__(self, parent, colors, cmap_name: str, cmap_values: np.ndarray):
-        super(ColorFeature, self).__init__(parent, colors)
```

Comment:
This case was a potential bug because it'd skip the `ColorFeature.__init__`. Did not yet result in a problem yet, because that init does nothing.

Reply:
Or was this intentional?

Reply:
Looks like it is.